chain-connect/src/GalachainConnectClient.ts

```diff
@@ -26,12 +26,16 @@ export class GalachainConnectClient {
   #provider: BrowserProvider | undefined;
   #chainCodeUrl: string;
 
-  get address() {
+  get ethAddress() {
     return `eth|${this.#address}`;
   }
 
+  get galachainAddress() {
+    return getAddress(this.#address);
+  }
+
   set address(val: string) {
```

Comment:
Since you're saving an ethereum address here, I would change this to `set ethAddress(val: string)...`